hack/common.sh

```diff
@@ -19,3 +19,8 @@ export OCP_VERSION="${OCP_VERSION:-4.8}"
 
 export TESTS_REPORTS_PATH="${TESTS_REPORTS_PATH:-/tmp/artifacts/}"
 
+# expect oc to be in PATH by default
+export OC_TOOL="${OC_TOOL:-oc}"
+
+export CONTAINER_MGMT_CLI="${CONTAINER_MGMT_CLI:-docker}"
```

Comment:
@fedepaol 
probably about time to change default to podman :-)

Reply:
Lol I was young(er) :P